packages/mobile/src/fiatExchanges/utils.tsx

```diff
@@ -59,6 +89,152 @@ export const fetchLocationFromIpAddress = async () => {
   return ipAddressObj
 }
 
+export const fetchSimplexQuote = async (
+  userAddress: string,
+  ipAddress: string,
+  currencyToBuy: CurrencyCode,
+  fiatCurrency: LocalCurrencyCode,
+  fiatAmount: number
+) => {
+  const response = await fetch(`${networkConfig.simplexUrl}/wallet/merchant/v2/quote`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      Authorization: `ApiKey ${networkConfig.simplexApiKey}`,
+    },
+    body: JSON.stringify({
+      end_user_id: userAddress,
+      digital_currency: currencyToBuy,
+      fiat_currency: fiatCurrency,
+      requested_currency: fiatCurrency,
+      requested_amount: fiatAmount,
+      wallet_id: 'valorapp',
+      client_ip: ipAddress,
+      payment_methods: ['credit_card'],
```

Comment:
Is this on purpose? I know we usually prioritise debit cards and bank transfers over credit, so just want to make sure this is intended.

Reply:
Yes it's intended. Simplex only allows cards and "simplex accounts" and we don't think enabling "simplex accounts" is worth the added complexity